controller/api/destination/watcher/server_watcher.go

```diff
@@ -36,12 +41,21 @@ type ServerUpdateListener interface {
 	UpdateProtocol(bool)
 }
 
+var serverMetrics = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "server_port_subscribers",
+		Help: "Number of subscribers to changes to Servers associated to a port in a pod.",
```

Comment:
What do you think of wording it this way?

```sugggestion
		Help: "Number of subscribers to Server changes associated with a pod's port.",
```

Reply:
:+1: